compiler/sem/sigmatch.nim

```diff
@@ -2214,7 +2219,7 @@ proc instantiateRoutineExpr(c: PContext, bindings: TIdTable, n: PNode): PNode =
       else:
         newSymNode(inferred, n.info)
   of nkProcTy, nkIteratorTy:
-    # possible in a concept context. There's nothing to instantiate
+    # possibly in a concept context. There's nothing to instantiate
```

Comment:
What I meant to convey here is that `nkProcTy` and `nkIteratorTy` are possible *only* in a concept context; "possibly" - to me - reads like both nodes can also appear outside of a concept context (which they can't).

(Not a blocker, I just wanted to provide some background on what the intended meaning is/was.)

Reply:
Ah, got it. Thanks for the explanation.